frontend/package.json

```diff
@@ -194,6 +196,7 @@
     "throttle-debounce": "2.1.0",
     "timer-machine": "1.1.0",
     "uuid": "3.3.2",
-    "webpack-git-hash": "1.0.2"
+    "webpack-git-hash": "1.0.2",
+    "why-did-you-update": "^1.0.8"
```

Comment:
Isn't that one deprecated ? I recall we switched to https://github.com/metasfresh/metasfresh/blob/master/frontend/package.json#L64